app/tests/api/inventory.test.ts

```diff
@@ -110,6 +175,43 @@ describe("Inventory API", () => {
     assert.equal(data.totalEmissions, inventory.totalEmissions);
   });
 
+  it("should find an inventory with csv format", async () => {
+    const url = `http://localhost:3000/api/v0/city/${locode}/inventory/${inventory.year}?format=csv`;
+    const req = createRequest(url);
+    const res = await findInventory(req, {
+      params: { city: locode, year: inventory.year.toString() }
+    });
+    assert.equal(res.status, 200);
+    assert.equal(res.headers.get("content-type"), "text/csv");
+    assert.ok(res.headers.get("content-disposition")?.startsWith("attachment"));
+    const csv = await res.text();
+    const lines = csv.split("\n");
+    assert.ok(lines.length > 0);
+    const headers = lines[0].split(",");
+    assert.equal(headers.length, 6);
+    assert.equal(headers[0], "Inventory Reference");
+    assert.equal(headers[1], "Total Emissions");
+    assert.equal(headers[2], "Activity Units");
+    assert.equal(headers[3], "Activity Value");
+    assert.equal(headers[4], "Emission Factor Value");
+    assert.equal(headers[5], "Datasource ID");
```

Comment:
You can test all of this in one assertion with `assert.deepEqual(headers, ["Inventory Reference", ...]);`, which will also have more helpful error messages.